src/pages/LandingPage/LandingPage.scss

```diff
@@ -1,14 +1,26 @@
+@import "styles/mixins.scss";
+@import "styles/mediaQueries.scss";
+
 .landing-page {
   height: 100vh;
   background-size: cover;
-  background-position: center;
+  background-position: right center;
```

Comment:
Should we do this with tailwind?

https://tailwindcss.com/docs/background-position/

Reply:
Yeah I can change this

Reply:
Done